controllers/control-plane/kcp_suite_test.go

```diff
@@ -163,6 +162,5 @@ var _ = AfterSuite(func() {
 
 	err := controlPlaneEnv.Stop()
 	Expect(err).NotTo(HaveOccurred())
-	err = runtimeEnv.Stop()
-	Expect(err).NotTo(HaveOccurred())
+
```

Comment:
All is well just this new line, could you please remove it? Many thanks for the fix!